vaadin-select-flow/pom.xml

```diff
@@ -46,28 +46,28 @@
         <dependency>
             <groupId>org.webjars.bowergithub.vaadin</groupId>
             <artifactId>vaadin-select</artifactId>
-            <version>2.1.7</version>
+            <version>2.2.0</version>
         </dependency>
         <!-- Lock transitive webjar dependencies for V13 versions-->
         <dependency>
             <groupId>org.webjars.bowergithub.vaadin</groupId>
             <artifactId>vaadin-overlay</artifactId>
-            <version>3.2.12</version>
+            <version>3.4.0</version>
         </dependency>
         <dependency>
             <groupId>org.webjars.bowergithub.vaadin</groupId>
             <artifactId>vaadin-text-field</artifactId>
-            <version>2.4.7</version>
+            <version>2.6.0</version>
         </dependency>
         <dependency>
             <groupId>org.webjars.bowergithub.vaadin</groupId>
             <artifactId>vaadin-item</artifactId>
-            <version>2.1.1</version>
+            <version>2.2.0</version>
         </dependency>
         <dependency>
             <groupId>org.webjars.bowergithub.vaadin</groupId>
             <artifactId>vaadin-list-box</artifactId>
-            <version>1.1.1</version>
+            <version>1.3.0</version>
         </dependency>
         <dependency>
             <groupId>org.webjars.bowergithub.vaadin</groupId>
```

Comment:
`list-mixin` should be bumped as it includes `rtl` functionality.

Reply:
Good catch. Fixed.